lib/documents/schemas/asylum_support_decisions.json

```
@@ -160,6 +160,10 @@
           "label": "Bayati, C",
           "value": "bayati-c"
         },
+    {
+      "label": "Beg, S",
+      "value": "beg-s"
+    },
```

I should probably not be such a :neckbeard: about this, but it looks like you want 8 spaces, not 4...

```suggestion
        {
          "label": "Beg, S",
          "value": "beg-s"
        },
```

Happy to approve it anyway though.

---

Not pedantry - holding me up to the right standards, and thanks!